qa/common/shm_util.py

```diff
@@ -52,15 +53,26 @@ def create_set_shm_regions(input0_list, input1_list, output0_byte_size,
                            output1_byte_size, outputs, shm_region_names,
                            precreated_shm_regions, use_system_shared_memory,
                            use_cuda_shared_memory):
+    from infer_util import get_number_of_bytes_for_npobject
```

Comment:
Why not include this at the top? after `from tritonclient.utils import *`

Reply:
It creates a circular dependency. infer_util -> shm_util -> infer_util

Reply:
All the more reason for it to be moved to tritonclient.utils

Reply:
I moved it to the tritonclient.utils.